plutus-core/cost-model/budgeting-bench/Bench.hs

```diff
@@ -220,6 +220,20 @@ benchSameTwoByteStrings :: DefaultFun -> Benchmark
 benchSameTwoByteStrings name = createTwoTermBuiltinBenchElementwise name (byteStringsToBench seedA)
                                ((\(bs, e) -> (BS.copy bs, e)) <$> byteStringsToBench seedA)
 
+benchIndexBytestring :: StdGen -> Benchmark
+benchIndexBytestring gen = createTwoTermBuiltinBenchElementwise IndexByteString (byteStringsToBench seedA) numbers
+    where
+        numbers = map (\s -> let x = fst $ randomR (0, s - 1) gen in (x, memoryUsage x)) byteStringSizes
+
+benchSliceByteString :: StdGen -> Benchmark
+benchSliceByteString gen = bgroup (show SliceByteString) $
```

Comment:
This looks basically OK.  I think I'll have to do some experimentation to see how it works in practice, but let's merge now and worry about that later.  I'll also add the appropriate R code and so on.